releasenotes/notes/discrete-pulse-library-deprecation-3a95eba7e29d8d49.yaml

```diff
@@ -0,0 +1,40 @@
+---
+features:
+  - |
+    The ``SymbolicPulse`` library was extended. The new pulses in the library are:
+
+      * :func:``~qiskit.pulse.library.GaussianDeriv``
+      * :func:``~qiskit.pulse.library.Sech``
+      * :func:``~qiskit.pulse.library.SechDeriv``
+      * :func:``~qiskit.pulse.library.Square``
+
+    The new functions return a ``ScalableSymbolicPulse``, and match the functionality
+    of the corresponding functions in the discrete pulse library, with the exception of
+    ``Square()`` which uses the convention ``sign(0)=0`` instead of ``sign(0)=1`` in
+    the discrete ``square()``.
+deprecations:
```

Comment:
Probably this note should go to 0.26. According to the deprecation section guide in [reno documentation](https://docs.openstack.org/reno/2.4.1/usage.html#editing-a-release-note),

> A list of features, APIs, configuration options to be deprecated in the release. Deprecations should not be used for something that is removed in the release, use upgrade section instead. Deprecation should allow time for users to make necessary changes for the removal to happen in a future release.

Reply:
You're right. I'll remove it. Thanks.